### PR TITLE
:memo: The videos listed in the README have been replaced with the latest version.

### PR DESCRIPTION
**Pull request by Corvus400**
## Issue
- None.

## Link
- https://github.com/Corvus400/conference-app-2023/tree/docs/update_readme_video

## Overview (Required)
- The title says it all.

### CODE REVIEW

**Comment by Corvus400** [comment]
Do you want to post a video of Light mode as well? 🤔

File: README.md [hunk 1]
```diff
@@ -12,7 +12,9 @@
 
 This is a video of an app in development, and it will be updated as needed.
 
-[Screen_recording_20230805_160952.webm](https://github.com/DroidKaigi/conference-app-2023-images/assets/1386930/32968ea9-6ef8-4707-b402-0fd009cc04fe)
+<video src="https://github.com/DroidKaigi/conference-app-2023/assets/13657682/ec91d1b8-639d-4622-8371-1387be4c6bac" width="320" controls>
```
Comment (takahirom) [inline README.md:15]: Just for your reference, you can use demo mode in OS's developers option to fix the time and status bar

**Review by takahirom — APPROVED** [review]
I've commented but it is better than what we have! Thanks

File: README.md (hunk 1 above)
Comment (Corvus400) [reply README.md:15]: I did not know this!
Thanks for letting us know.

**Pull request merged**